### PR TITLE
all: remove builder code, update README.md

### PR DESCRIPTION
**Pull request by danderson**
The synology packages now get built by the same build machinery as our other packages (debian, rpm, etc.) in github.com/tailscale/tailscale.

Keep the README with pointers on how to install, where to file issues, how to build from source, and so forth. This should keep incoming links from things like old tutorials alive.

Fixes tailscale/tailscale#8217

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,53 @@
...
 4.  Follow the wizard until done.
 5.  At this point `tailscaled` should be up and running.
-6.  SSH into the  machine, and run `sudo tailscale up` so you can authenticate.
-
-## Upgrading
-
-If upgrading to version v1.10.0, you may end up with duplicate installations of Tailscale. This is a [known](https://github.com/tailscale/tailscale/issues/2266#issuecomment-869792505) side effect of some metadata changes that were made in v1.10.0 in preparation of the installation package to be listed in the Synology Package Center. It is recommended to uninstall the old Tailscale package first before upgrading to v1.10.0. Please note that your devices Tailscale IP may change when v1.10.0 is installed.
-
-## Compatibility
+6.  SSH into the machine, and run `sudo tailscale up` so you can
```
Comment (DentonGentry) [inline README.md:17]: ...I guess we haven't updated this in a while. They don't need to ssh now, they click on the Tailscale icon in the Package Center and then click "open" to be taken to a web page where they can authenticate.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
-If you need to _**compile**_ a synology package, check it out.
+- Thanks to [@nirev](https://github.com/nirev) for creating this
+  project and transferring it to Tailscale's GitHub org.
+- https://haugene.github.io/docker-transmission-openvpn/synology-nas/
+  for the /dev/net/tun thing
+- Package structure: [Synology Package Developer
+  Guide](https://help.synology.com/developer-guide/index.html)
+- The package building process was originally based on
+  [synology-wireguard](https://github.com/runfalk/synology-wireguard). If
+  you need to _**compile**_ a synology package, check it out.
```
Comment (DentonGentry) [inline README.md:53]: "If you need to _**compile**_ a synology package, check it out."
Probably not applicable any more.

**Review by DentonGentry — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (danderson) [reply README.md]: I replaced this entire section with a link to https://tailscale.com/kb/1131/synology/ , which is more up to date and also covers installation via the official packaging center.

File: README.md (hunk 2 above)
Comment (danderson) [reply README.md]: This is in the credits from the original author, citing the things they found useful. Reworded to clarify that it's handy for anyone who needs to cross-compile C code for Synology devices, unrelated to our own package.

**Review by DentonGentry — APPROVED** [review]

**Pull request merged**